ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86-loaded-upgrade-from-4.14.yaml

```diff
@@ -40,8 +40,10 @@ tests:
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
       CHURN: "false"
+      CONTROL_PLANE_INSTANCE_TYPE: m6i.4xlarge
```

Comment:
m6i.4xlarge? Is this intended change? Seems much larger than previous releases.

Reply:
This change should by some failure of cluster-density-heavy, but it seems it's not relate with the instance type, restore to default now